### PR TITLE
Chords above text

### PR DESCRIPTION
**Pull request by anniviz**
the chords are now shown on top of the lyrics

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,31 +1,40 @@
...
+    gridTemplateColumns: isAllSongsShown ? '60px 25% auto' : '60px 0% auto',
+  })
+
   return (
-    <Layout>
+    <AnimatedLayout style={animateLayout}>
       <Sidebar>
         <SidebarItem>
-          <GradientText>Current Set List</GradientText>
+          <GradientText>{/* Current Set List */}</GradientText>
```
Comment (maje02) [inline src/App.js]: delete comment

File: src/App.js [hunk 2]
```diff
@@ -1,31 +1,40 @@
...
         </SidebarItem>
         <SidebarItem>
-          <GradientText>Set Lists</GradientText>
+          <GradientText>{/* Set Lists */}</GradientText>
```
Comment (maje02) [inline src/App.js]: delete comment

File: src/Song/Song.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+export default function Song({ song, isAListShown }) {
+  return (
+    <SongWrapper isAListShown={isAListShown}>
+      <SongTitle>{song.optimizedMetaData.title}</SongTitle>
+      {song.optimizedMetaData.artist && (
+        <SongArtist>{song.optimizedMetaData.artist}</SongArtist>
+      )}
+      <LyricsWrapper>
+        {song.lines.map((line, index) => {
+          const areThereChords = areChordsInLine(line.items)
```
Comment (maje02) [inline src/Song/Song.js]: improve naming

File: src/Song/Song.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    return items.some(item => item._name)
+  }
+
+  function areChordsInLine(items) {
+    return items.some(item => item.chords)
+  }
+}
+
+const SongWrapper = styled.section`
+  /* grid-column: ${props => (props.isAListShown ? '3 / end' : '2 / end')}; */
```
Comment (maje02) [inline src/Song/Song.js]: delete comment

File: src/Song/ChordLyricsPair.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  height: 1em;
+  margin: 4px;
+  color: #939fc2;
+`
+
+const Lyrics = styled.div`
+  color: #fefefe;
+  margin-bottom: 4px;
+  line-height: 1.5;
+`
```
Comment (maje02) [inline src/Song/ChordLyricsPair.js:30]: Add prop-types

File: src/TitleList/TitleList.js [hunk 6]
```diff
@@ -13,9 +13,10 @@ export default function TitleList({
 }) {
   const AnimatedTitleWrapperBorder = animated(TitleWrapperBorder)
   const flyIn = useSpring({
-    width: isAllSongsShown ? '100%' : '0%',
-    transform: isAllSongsShown ? 'scale(1)' : 'scale(0.6)',
-    display: isAllSongsShown ? 'block' : 'none',
+    // width: isAllSongsShown ? '100%' : '0%',
+    // transform: isAllSongsShown ? 'scale(1)' : 'scale(0.6)',
+    // display: isAllSongsShown ? 'block' : 'none',
```
Comment (maje02) [inline src/TitleList/TitleList.js]: delete comments

**Review by maje02 — CHANGES_REQUESTED** [review]
change names and delete comments

File: src/App.js (hunk 1 above)
Comment (anniviz) [reply src/App.js]: done

File: src/App.js (hunk 2 above)
Comment (anniviz) [reply src/App.js]: done

File: src/Song/Song.js (hunk 3 above)
Comment (anniviz) [reply src/Song/Song.js]: done

File: src/Song/Song.js (hunk 4 above)
Comment (anniviz) [reply src/Song/Song.js]: done

File: src/Song/ChordLyricsPair.js (hunk 5 above)
Comment (anniviz) [reply src/Song/ChordLyricsPair.js:30]: done

File: src/TitleList/TitleList.js (hunk 6 above)
Comment (anniviz) [reply src/TitleList/TitleList.js]: done

File: src/Song/ChordLyricsPair.js [hunk 7]
```diff
@@ -11,7 +12,7 @@ export default function ChordLyricsPair({ item, chordsInLine }) {
 }
 
 const ChordLyricsWrapper = styled.div`
-  margin-right: 4px;
+  /* margin-right: 4px; */
```
Comment (maje02) [inline src/Song/ChordLyricsPair.js]: delete comment

**Review by maje02 — APPROVED** [review]
there is still one comment in ChordLyricsPair.js, but the rest is great :)

**Pull request merged**